packages/shared/src/components/modals/CompanionModal.tsx

```diff
@@ -39,105 +41,108 @@ export default function CompanionModal({
       kind={Modal.Kind.FlexibleCenter}
       size={size}
     >
-      {isTablet && (
-        // eslint-disable-next-line jsx-a11y/media-has-caption
-        <video
-          className="tablet:hidden rounded-16 w-fit"
-          poster={cloudinary.companion.videoThumbnailMobile}
-          loop
-          autoPlay
-        >
-          <source
-            src={cloudinary.companion.backgroundVideoMobile}
-            type="video/mp4"
-          />
-        </video>
-      )}
+      <Modal.Body className={classNames('p-0', { 'min-h-[640px]': !loaded })}>
```

Comment:
Can't we rather mock the video size, the body should auto adjust.
Alternative we can wait before render?

In anycase we don't use px values ever, always use REM when you need.
(Expection is a 1px line)